.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+<!--- 
+Replace this comment with a description of the problem.
+
+Please help us by providing all of the details listed below. A complete and thoughtful issue request with research on possible root causes and diagnostics goes a very long way in helping us rapidly get your issues resolved. 
+--> 
+
+**Reproduction Steps:**
+
+**OS and version:**    
+
```

Comment:
Maybe more explicit language is helpful:
- are you running on minishift or something else
- if minishift include output of `minishift version`
- include output of `oc version`
- rhche version / does it reproduce on latest official image